### ISSUE TITLE
use named options for hook

### ISSUE THREAD

**Issue by zhaoyao91** [karl-run/react-bottom-scroll-listener#34]
currently the hook is using positioned options, which are hard to specify only wanted override since there many options

**Comment by karl-run** [comment]
That's a really good point. Should have taken that into consideration when I made the hook.

**Comment by karl-run** [comment]
The signature is now as follows:

```
function useBottomScrollListener<T extends HTMLElement>(
  onBottom: () => void,
  options?: {
    offset?: number;
    debounce?: number;
    debounceOptions?: DebounceOptions;
    triggerOnNoScroll?: boolean;
  },
): RefObject<T> {
```

and is published as v5.0.0. 

Please give it a try. :)
